turbopack/crates/turbopack-core/src/compile_time_info.rs

```diff
@@ -238,6 +238,8 @@ impl From<CompileTimeDefineValue> for FreeVarReference {
     }
 }
 
+/// Like [CompileTimeDefines], these are used for constant evaluation. But additionally, any such
```

Comment:
> these are used for constant evaluation

are they?

Reply:
So apparently free_vars only for typeofs, should that be removed?
https://github.com/vercel/next.js/blob/51a62c50062e3c86e32b15a26c3fe69fd51208a0/turbopack/crates/turbopack-ecmascript/src/references/mod.rs#L2841-L2856